client/src/components/profile/info-change.js

```diff
@@ -113,14 +125,14 @@ class InfoChange extends React.Component {
           <Col md="6">
             <Input
               name="name"
-              placeholder={user.name}
```

Comment:
You should left placeholder with information messages, like "Put your name"

Reply:
Fixed